File: src/main/java/org/noxet/noxetserver/messaging/NoteMessage.java
package org.noxet.noxetserver.messaging;

import net.md_5.bungee.api.ChatColor;

public class NoteMessage extends Message {
    public NoteMessage(String text) {
        super(text);
    }

    @Override
    public ChatColor getDefaultColor() {
        return ChatColor.LIGHT_PURPLE;
    }

    @Override
    public String getPrefix() {
        return super.getPrefix() + "§5Note: ";
    }
}

File: src/main/java/org/noxet/noxetserver/messaging/NoteMessage.scala
package org.noxet.noxetserver.messaging;

import net.md_5.bungee.api.ChatColor;

class NoteMessage(text: String) extends Message(text):
    override def getDefaultColor: ChatColor = ChatColor.LIGHT_PURPLE
    override def getPrefix: String = super.getPrefix + "§5Note: "
    